lib/pack.js

```diff
@@ -61,31 +76,30 @@ module.exports = {
     if (!providerRuntime && !funcRuntime) {
       console.error('No runtime found at global provider or local function level, eg. dotnetcore2.1')
     }
-  
+
     return (providerRuntime && providerRuntime.startsWith('dotnet') && !funcRuntime)
-        || (funcRuntime && funcRuntime.startsWith('dotnet'))
-        || (!providerRuntime && funcRuntime && funcRuntime.startsWith('dotnet'));
+      || (funcRuntime && funcRuntime.startsWith('dotnet'))
+      || (!providerRuntime && funcRuntime && funcRuntime.startsWith('dotnet'));
   },
 
   pack() {
     let cli = this.serverless.cli;
+    let flags = getLambdaPackageFlags(this.serverless.service)
     cli.log('Serverless DotNet: Pack');
-
     const dicArtifacts = this.getPackingInfo();
-
     var promises = Object.entries(dicArtifacts).map(kv => {
       return new BbPromise(function (resolve, reject) {
 
         try {
           const packingInfo = kv[1];
 
-          var output=program.execSync('dotnet lambda package -o ' + packingInfo.artifactOutput, { "cwd": packingInfo.currentWorkingDir }, function (error, stdout, stderr) {
-            //console.log(stdout);
+          var output = program.execSync('dotnet lambda package ' + flags + ' -o ' + packingInfo.artifactOutput, { "cwd": packingInfo.currentWorkingDir }, function (error, stdout, stderr) {
+            console.log(output)
```

Comment:
You like to always have this in output?

Reply:
Yes, it will be an empty an empty string in case there are no values needed for runtime or architecture